### DISCUSSION TITLE
Failing to setup allure reporting with ruby cucumber project

### DISCUSSION THREAD

**Discussion by nicholaspshaw** [allure-framework/allure2#2038] [Questions & Support] [answered]
Hello,

I'm trying to get allure reporting setup with our project and thus far it doesn't appear to be going great.  I wanted to prove this out locally before moving on to CI.

I have followed the instructions here: https://docs.qameta.io/allure-report/#_cucumber and here: https://github.com/allure-framework/allure-ruby/blob/master/allure-cucumber/README.md

Versions:
  gem 'allure-cucumber', '~> 2.18.0'
  gem 'cucumber-rails', '~> 2.4',
  cucumber 8 being used.
    
The command I'm generally using from my rubymine IDE is:

--color -r features --exclude special_tests --format AllureCucumber::Formatter -- out  /tmp/reports

I get the following error:

uninitialized constant AllureCucumber::Formatter
Error creating formatter: AllureCucumber::Formatter (NameError)
/Users/me/.rbenv/versions/3.0.2/gemsets/proj/gems/cucumber-8.0.0/lib/cucumber/constantize.rb:40:in `constantize_name'

Oddly if I change the command slightly and drop the --out param, which should mean it uses what is in the config, it runs successfully via the IDE at least but doesn't produce a report in the expected location

Can anyone point me in a useful direction?  I'm a bit clueless as it stands.

**Comment by andrcuns** [comment, marked as answer]
@nicholaspshaw class name `AllureCucumber::Formatter` is wrong, it doesn't exist. Check the example from https://github.com/allure-framework/allure-ruby/blob/master/allure-cucumber/README.md#usage.

There is also example project with full setup: https://github.com/allure-examples/allure-cucumber-example
